tox.ini

```diff
@@ -27,7 +27,8 @@ deps =
 commands =
     sh -c 'kolibri manage makemigrations --check'
     # Run the actual tests
-    python -O -m pytest {posargs:--cov=kolibri --cov-report= --cov-append --color=no}
+    python -O -m pytest {posargs:--cov=kolibri --cov-report= --cov-append --color=no} kolibri
+    python -O -m pytest {posargs:--cov=kolibri --cov-report= --cov-append --color=no} -p no:django test
```

Comment:
My initial concern was that the second execution would overwrite the code coverage, but seems that's what `--cov-append` addresses.

Reply:
We're also not actually reporting our code coverage - it gets messed up because of our conditional test runs.